src/components/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Box, SimpleGrid, useTheme } from '@chakra-ui/core';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+
+import Card, { CardSizes } from '../Card/Card';
+import StoryHeading from '../StoryHeading/StoryHeading';
+import Helper from '../Helper';
+import Icon, { IconSize } from './Icon';
+import Input from '../Input';
+import Label from '../Label';
+
+type IconsFiltererProps = {
+  children: (icons: string[]) => React.ReactNode;
+  filter?: string;
+};
+
+const IconsLister = ({ children, filter = '' }: IconsFiltererProps) => {
```

Comment:
For some reasons, the `useTheme` hook wasn't returning the theme inside a story 🤷

Reply:
is the story wrapped in a ThemeProvider?

Reply:
Yes, cf: https://github.com/heetch/flamingo/pull/158/files#diff-41e445b678933963d77632d4206c90b8R17
It doesn't work only when the hook is used at the top level 🤷